crates/sui-node/src/lib.rs

```diff
@@ -152,21 +158,34 @@ impl SuiNode {
             tokio::spawn(server.serve().map_err(Into::into))
         };
 
-        let json_rpc_service = if config.consensus_config().is_some() {
-            None
+        let (json_rpc_service, ws_subscription_service) = if config.consensus_config().is_some() {
+            (None, None)
```

Comment:
this is not the optimal flag to use IMO ... but we can leave it for another day. I will create an issue to track